test/production/custom-server/custom-server.test.ts

```diff
@@ -15,6 +15,10 @@ createNextDescribe(
       expect($('p').text()).toBe(`Page ${page}`)
     })
 
+    it('should not log any error messages when server is started with "quiet" setting', async () => {
+      await next.render(`/error`)
+    })
```

Comment:
@ijjk My first impl. was here: https://github.com/vercel/next.js/blob/canary/test/integration/custom-server/test/index.test.js That was super simple. But not for production mode :(

Reply:
They can be set via the `env` field next to `startCommand` then the CLI output can be accessed via `next.cliOutput`.

Reply:
@ijjk Much appreciated for the help. I added two test cases:

- `quite: true` -> no logging
- default `quite` -> logging